sdk/azcore/request.go

```diff
@@ -66,24 +66,34 @@ func (ov opValues) get(value interface{}) bool {
 	return ok
 }
 
+func stripQueryParams(path string) (string, string) {
```

Comment:
If this function is only ever called from inside JoinPaths, I'd move it in there. In fact, I probably wouldn't have it be a function and just move the code inside JoinPaths.